WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -319,6 +319,7 @@ - (void)tableView:(UITableView *)tableView
                 WPAccount *defaultAccount = [accountService defaultWordPressComAccount];
 
                 if (!defaultAccount) {
+                    [WPAnalytics track:WPAnalyticsStatLogout];
```

Comment:
Code looks ok, but I find it that we're logging the user out in weird places in our code.  This is one of those cases.